### ISSUE TITLE
False Positive on MD046

### ISSUE THREAD

**Issue by 0xdf223** [rvben/rumdl#31]
If a file has both codeblocks (fenced) and inline HTML, it will incorrectly identfy that HTML as a codeblock (indented). md file attached as the HTML tries to render inline.

[test5.md](https://github.com/user-attachments/files/21665802/test5.md)

```
$ rumdl check test5.md -e MD046
test5.md:13:1: [MD046] Use fenced code blocks [*]

Issues: Found 1 issues in 1 file (5ms)
Run with `--fix` to automatically fix 1 of the 1 issues
```

**Comment by rvben** [comment]
Hi @0xdf223,

Thanks for reporting this issue! This has been fixed in commit 80aa2b1 and is included in release v0.0.110.

Edit: expected in release 0.0.109.

**Comment by 0xdf223** [comment]
Probably a dumb question. When / where do I find 0.0.110? The latest I (and uv) see is 0.0.108.

**Comment by rvben** [comment]
Not a dumb question! It was my mistake actually, I thought 0.0.110 was the next release and I was planning to release that this morning. Turns out it should be 0.0.109 and I ran into some issues. Fixes on one side led to some issues in another rule/test.
